spec/integration/alert_mgmt_flows_spec.rb

```diff
@@ -68,10 +68,8 @@ def alerts_client
 
       # Verify trigger condition was updated in Hawkular
       trigger = alerts_client.get_single_trigger(hawkular_alert_id, true)
-      expect(trigger.conditions.count).to eq(2)
-
-      updated_condition = trigger.conditions.find { |c| c.operator == 'GT' }
-      expect(updated_condition.data2_multiplier).to eq(0.5)
+      expect(trigger.conditions.count).to eq(1)
```

Comment:
out of curiosity, is know why this changed from 2 to 1?

Reply:
it's because in the past we used two conditions to test each threshold, with prometheus it is condensed into a single promql expression, in a single ExternalCondition.